pkg/builder/builder.go

```diff
@@ -845,16 +854,17 @@ func makeScrutinizeInitContainer(vscr *v1beta1.VerticaScrutinize, vdb *vapi.Vert
 		Env:          buildCommonEnvVars(vdb),
 	}
 	cnt.Env = append(cnt.Env, buildNMATLSCertsEnvVars(vdb)...)
+	cnt.Env = append(cnt.Env, buildScrutinizeTarballEnvVar(tarballName))
```

Comment:
minor: could we rewrite this to combine into a single append?
```suggestion
	cnt.Env = append(cnt.Env, buildScrutinizeTarballEnvVar(tarballName),
	     buildNMATLSCertsEnvVars(vdb)...)
```

Reply:
This syntax is not correct. How about:
```
cnt.Env = append(cnt.Env, append(buildNMATLSCertsEnvVars(vdb),
		buildScrutinizeTarballEnvVar(tarballName))...)
```